compose/services.yml

```diff
@@ -61,7 +61,7 @@ services:
     extends:
       file: ./common.yml
       service: component
-    image: balena/open-balena-registry:${OPENBALENA_REGISTRY_VERSION_TAG:-master}
+    image: balena/open-balena-registry:${OPENBALENA_REGISTRY_VERSION_TAG}
```

Comment:
You need to bump the registry version for minio/S3 to work properly.

Reply:
OK - we should make that part of this PR then

Reply:
bumped to the latest 👍

Reply:
bumped, but will need bumping again due to a PR which is needed to make S3 work from local environment-configured `confd` templates. See https://github.com/balena-io/open-balena-registry/pull/83